### PR TITLE
Feature: Typeahead for Predefined Property ComboBox

### PR DESCRIPTION
**Pull request by blankse**
Depends on https://github.com/pimcore/pimcore/pull/11141

### CODE REVIEW

File: bundles/AdminBundle/Controller/Admin/ElementController.php [hunk 1]
```diff
@@ -784,19 +784,20 @@ public function getPredefinedPropertiesAction(Request $request)
...
-                if (is_array($row['ctype'])) {
-                    $row['ctype'] = implode(',', $row['ctype']);
+            $list->setFilter(function (Model\Property\Predefined $predefined) use ($type, $query) {
+                if (!str_contains($predefined->getCtype(), $type)) {
+                    return false;
                 }
-                if (strpos($row['ctype'], $type) !== false) {
-                    return true;
+                if ($query && !str_contains($this->trans($predefined->getName()), $query)) {
+                    return false;
```
Comment (NiklasBr) [inline bundles/AdminBundle/Controller/Admin/ElementController.php:797]: `str_contains` is case-sensitive, this filter might be improved by making it non-sensitive

Comment (blankse) [reply bundles/AdminBundle/Controller/Admin/ElementController.php:797]: @NiklasBr You are right. I changed it to stripos https://github.com/pimcore/pimcore/pull/11142/commits/8296fc2e1936e839062e78a74c50beaa7ce4a629

**Comment by blankse** [comment]
I rebased the branch after merging #11141

**Comment by blankse** [comment]
@brusch @fashxp Can this be part of v10.3? Should be fast to review. We have a large amount of predefined properties. Would be a nice feature to can select this by typing.

**Comment by brusch** [comment]
Thanks!

**Comment by blankse** [comment]
@brusch Thanks 👍🏼

**Pull request merged**